AppCenter/AppCenter/Internals/Util/MSEncrypter.m

```diff
@@ -171,4 +217,22 @@ + (NSData *)generateKeyWithTag:(NSString *)keyTag {
   return resultKey;
 }
 
++ (NSData *)generateInitializationVector {
```

Comment:
Can this can be parametrized and re-used in `generateAndSaveKeyWithTag` since the only difference is amount of bytes?
```obj-c
+ (NSData *)generateAndSaveKeyWithTag:(NSString* )keyTag {
 NSData *resultKey = [self generateInitializationVector:kMSEncryptionKeySize];

 // Save key to the Keychain.
 NSString *stringKey = [resultKey base64EncodedStringWithOptions:0];
 [MSKeychainUtil storeString:stringKey forKey:keyTag];
 return resultKey;
}

+ (NSData *)generateInitializationVector {
 return [self generateInitializationVector:kCCBlockSizeAES128];
}

+ (NSData *)generateInitializationVector:(NSUInteger)keySize {
```

Reply:
We could, but this would slightly obfuscate things and there is not much similarity between the key and the initialization vectors at a conceptual level. It is more of a coincidence that they are generated the same way. Resolving this but feel free to reopen if you want to discuss further.